analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/ConditionEvaluatesToConstantTest.cs

```diff
@@ -26,57 +26,52 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class ConditionEvaluatesToConstantTest
     {
-        private static readonly DiagnosticDescriptor[] OnlyDiagnostics = new[] { ConditionEvaluatesToConstant.S2583, ConditionEvaluatesToConstant.S2589 };
+        private readonly VerifierBuilder sonar = new VerifierBuilder<SymbolicExecutionRunner>().WithBasePath(@"SymbolicExecution\Sonar")
```

Comment:
wouldn't be more clear to user `sonarVerifier`? I believe that `sonar` in our context can mean anything.